tests_tf/test_attacks.py

```diff
@@ -45,5 +114,189 @@ def test_generate_np(self):
         self.assertTrue(np.allclose(perturbation_norm, self.attack.eps))
 
 
+class TestFastGradientMethod(CleverHansTest):
+    def setUp(self):
+        super(TestFastGradientMethod, self).setUp()
+        import tensorflow as tf
+
+        # The world's simplest neural network
+        def my_model(x):
+            W1 = tf.constant([[1.5, .3], [-2, 0.3]], dtype=tf.float32)
+            h1 = tf.nn.sigmoid(tf.matmul(x, W1))
+            W2 = tf.constant([[-2.4, 1.2], [0.5, -2.3]], dtype=tf.float32)
+            res = tf.nn.softmax(tf.matmul(x, W2))
+            return res
+
+        self.sess = tf.Session()
+        self.model = my_model
+        self.attack = FastGradientMethod(self.model, sess=self.sess)
+
+    def help_test_generate_np_gives_adversarial_example(self, ord=None):
```

Comment:
I think nosetests runs all methods that have "test" at a word boundary demarcated by underscores, but does not run all methods. So if you change the name to "help_check_generate_np_gives_adversarial_example" or similar it should be possible to remove the hack.